sources/constraint-declaration-validation.asciidoc

```diff
@@ -138,7 +138,7 @@ A future revision of this specification might define support for such usages of
 ----
 public class User {
 
-    // preferred style as of Jakarta {spec-name-bv} 2.0
+    // preferred style as of Jakarta Bean Validation 2.0
```

Comment:
Note as for this one and the one above, we could enable attribute replacement in the code instead.

For the XSD file, we really need to hardcode the value as it's supposed to be verbatim the content of the files.

Reply:
This is what I had intended to do. It just avoided my attention